Gemfile.lock

```diff
@@ -51,4 +59,4 @@ DEPENDENCIES
   cucumber
 
 BUNDLED WITH
-   1.10.3
+   2.4.6
```

Comment:
All changes to the gemfile are to work with newer bundle. The Circle CI image no longer works out the box with older Ruby.

Note, Ruby is a functional test dependency only. Has no bearing on consumption of Drafter.